doc/Witch/intro-to-witch.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Introduction to Witch
+---
+
+To popularise software verification, we must make proof writing less of
+a burden on you, the user. Currently, only specialists and a few others 
+can write proofs for their programs, even though you surely understand
+the domain and invariants of your projects -- otherwise you wouldn't
+be able to write any software at all.
+
+*Our hypothesis is that you, programmers, are
```

Comment:
```suggestion
*Our hypothesis is that programmers, are
```

Reply:
I'd prefer to keep "you" rather than "programmers"? I think it should sound like a direct conversation than some sort of impersonal technical piece. Idk, I think it feels better.

Reply:
@thealmarty what do you think?

Reply:
I think keeping the "you" is fine if that's what you prefer. It matches the style with the rest of the docs.